python/lancedb/table.py

```diff
@@ -75,7 +75,7 @@ def _to_record_batch_generator(data: Iterable, schema, on_bad_vectors, fill_valu
 
 class Table(ABC):
     """
-    A [Table](Table) is a collection of Records in a LanceDB [Database](Database).
+    A Table is a collection of Records in a LanceDB Database.
```

Comment:
Is there any reason they were here? These are interpreted as bad links in markdown. See - https://lancedb.github.io/lancedb/python/python/#table

Reply:
not sure. maybe the intention was to link to the docs for the appropriate classes?